qa/L0_metrics/histogram_metrics_test.py

```diff
@@ -40,31 +40,38 @@
 import test_util as tu
 
 MILLIS_PER_SEC = 1000
+FIRST_RESPONSE_HISTOGRAM = "nv_inference_first_response_histogram_ms"
 
 
 def get_histogram_metric_key(
     metric_family, model_name, model_version, metric_type, le=""
 ):
     if metric_type in ["count", "sum"]:
         return f'{metric_family}_{metric_type}{{model="{model_name}",version="{model_version}"}}'
+    elif metric_type == "bucket":
+        return f'{metric_family}_{metric_type}{{model="{model_name}",version="{model_version}",le="{le}"}}'
     else:
         return None
 
 
-class TestHistogramMetrics(tu.TestResultCollector):
+class HistogramMetricsTest(tu.TestResultCollector):
```

Comment:
Please keep `Test` prefix in class name, it's required to be discovered by pytest, which we are gradually moving towards using for it's automatic test reports: https://docs.pytest.org/en/7.1.x/getting-started.html#group-multiple-tests-in-a-class. If someone tries to call this test with `pytest` in the future, I think it will fail to find any tests and they may miss that.

Reply:
example pytest usage: https://github.com/triton-inference-server/server/blob/4af437e4902ac2b540727e4985620e019f063161/qa/L0_metrics/test.sh#L146

Reply:
Updated